packages/test/src/utils/index.js

```diff
@@ -0,0 +1,68 @@
+// For react-testing-library helpers, overrides, and utilities
+// All elements from react-testing-library can be imported from this wrapper.
+// See https://testing-library.com/docs/react-testing-library/setup for more info
+import React from "react";
+import {
+  render,
+  waitFor,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import { renderHook } from '@testing-library/react-hooks';
+import { createStore, applyMiddleware, combineReducers } from "redux";
+import { Provider } from "react-redux";
+import { MemoryRouter } from "react-router-dom";
+
+export class RTLUtils {
+  /**
+   * Create a React testing library utils for foreman
+   * @param {ReduxReducers} reducers reducers to apply
+   * @param {ReduxMiddlewares} middlewares middlewares to apply
+   */
+  constructor(reducers, middlewares, initialState = {}) {
+    const combinedReducers = combineReducers(...reducers);
+    this.store = createStore(
+      combinedReducers,
+      initialState,
+      applyMiddleware(...middlewares)
+    );
+  }
+  /**
+   * Renders testable component with redux and react-router
+   * @param  {ReactNode} component A react node to render
+   * @param {Object} options renderOptions and routerParams
+   * @return {Node}  rtl-rendered component
+   */
+  renderWithRedux(component, options) {
```

Comment:
Would it make sense to make it more extensible so that more wrappers can be optionally added if there is a need?

Intl[https://github.com/theforeman/foreman_ansible/pull/419/files#diff-12cc8acc33a4670102459dc74291e7818591f4d667dc9d3f6bb432a4cdaa4549R31] or context providers are likely candidates.